docs/embed-api.rst

```diff
@@ -0,0 +1,122 @@
+Embed API
+=========
+
+Read the Docs allows you to embed content from any of the projects we host.
+This allows reuse of content across sites, making sure the content is always up to date.
+
+.. contents::
+   :local:
+
+Usage
+-----
+
+Check our :doc:`/guides/embedding-content` guide to learn how to use the embed API in your projects.
+Additionally, you can use the API with:
+
+- `sphinx-hoverxref`_: Sphinx extension
+
+.. _sphinx-hoverxref: https://sphinx-hoverxref.readthedocs.io
+
+API
+---
+
+The embed API is exposed from the domain where your docs are being served.
+This is ``https://docs.readthedocs.io/_/api/v2/embed`` for the ``docs`` project, for example.
+
+.. warning::
+
+   This API isn't stable yet, feel free to `open an issue`_ if you find any problems.
+   Only Sphinx projects are supported at the moment,
+   a new version of this API with more general support is on the way!
+
+   .. _open an issue: https://github.com/readthedocs/readthedocs.org/issues/new
+
+.. http:get:: /_/api/v2/embed/
+
+   Return the content of a section with all links rewritten to be absolute.
+   There are two ways of using the API
+
+   Using the project and version slug.
+
+   .. Request
+
+   :query project: Project slug
+   :query version: Version slug
+   :query path: Path of file
+   :query section: Section ID
+
+   Using the URL.
+
+   :query url: Full URL with fragment
```

Comment:
It's not clear what "Full URL" the user has to put here. We should make it more clear to avoid confusion.

Also, I think fragment is optional.